src/components/Layout/Layout.jsx

```diff
@@ -1,16 +1,27 @@
 import React from "react";
 import Drawer from "./Drawer";
 import Header from "./Header";
+import Footer from "./Footer";
+import { Stack, styled } from "@mui/material";
 
-const Layout = () => {
+const Layout = ({ children }) => {
   const [open, setOpen] = React.useState(false);
```

Comment:
```suggestion
  const [isDrawerOpen, setIsDrawerOpen] = React.useState(false);
```
이런식으로 변수는 길어도 상관없으니 변수만 보고도 의미를 파악할 수 있도록 변경. 다른 부분할때도 참고.
이 부분 바뀌면 prop 넘겨주는 부분 명칭도 바뀌어야겠지?